test/version/version-external-packages.test.js

```diff
@@ -9,35 +9,41 @@ const cliPkgJSON = require('../../packages/webpack-cli/package.json');
 
 describe('version flag with external packages', () => {
     it('outputs version with init', () => {
-        const { stdout, stderr } = run(__dirname, ['init', '--version']);
+        const { stdout, stderr } = run(__dirname, ['init', '--version'], false);
```

Comment:
Why `false` at the end?

Reply:
without false it is `webpack init --version --output <path>`

Reply:
:+1:

Reply:
https://github.com/webpack/webpack-cli/blob/13bf7e5f84ff425886c39e058400823366261751/test/utils/test-utils.js#L18